src/components/v2v3/shared/V2V3ProjectHandle.tsx

```diff
@@ -20,16 +20,16 @@ export default function V2V3ProjectHandle({
 
   return (
     <Link href={v2v3ProjectRoute({ projectId, handle: handleToRender })}>
-      <a
-        style={{ fontWeight: 500, ...style }}
+      <div
```

Comment:
I think we tried this previously. The problem is that we do need an `a` tag here, because otherwise it breaks styling and accessibility => links should always be `a`.

I think the cause of the console error is that V2V3ProjectHandle is used somewhere within another `a`

Reply:
Got an idea